backend/src/main/java/com/google/rolecall/services/UserServices.java

```diff
@@ -96,21 +99,25 @@ public User editUser(UserInfo newUser) throws EntityNotFoundException, InvalidPa
     User.Builder builder = this.getUser(newUser.id()).toBuilder()
         .setFirstName(newUser.firstName())
         .setLastName(newUser.lastName())
+        .setEmail(newUser.email())
+        .setPhoneNumber(newUser.phoneNumber())
         .setDateJoined(newUser.dateJoined())
-        .setEmergencyContactName(newUser.emergencyContactName())
-        .setEmergencyContactNumber(newUser.emergencyContactNumber())
-        .setComments(newUser.comments())
-        .setRole(newUser.role())
-        .setIsActive(newUser.isActive())
+        .setIsAdmin(newUser.isAdmin())
+        .setIsCoreographer(newUser.isCoreographer())
+        .setIsDancer(newUser.isDancer())
+        .setIsOther(newUser.isOther())
         .setCanLogin(newUser.canLogin())
-        .setAdmin(newUser.admin())
         .setRecievesNotifications(newUser.notifications())
         .setManagePerformances(newUser.managePerformances())
         .setManageCasts(newUser.manageCasts())
         .setManagePieces(newUser.managePieces())
         .setManageRoles(newUser.manageRoles())
-        .setManageRules(newUser.manageRules());
-
+        .setManageRules(newUser.manageRules())
+        .setEmergencyContactName(newUser.emergencyContactName())
+        .setEmergencyContactNumber(newUser.emergencyContactNumber())
+        .setComments(newUser.comments())
+        .setIsActive(newUser.isActive())
+        ;
```

Comment:
```suggestion
        .setIsActive(newUser.isActive());
```

Reply:
Zephyr, the reason for placing the semicolon on a separate line is that the semicolon is the one asymmetry that makes it harder to add another field initialization after setIsActive(newUser.isActive());. If you try to, and forget to remove the semicolon, you get a syntax error. By moving the semicolon to a separate line, you can add initializers or move the initializers around freely without worrying where the semicolon is. For the same reason, I try to adda comma after every field definition in typescript, even after the last one that is optional.

Reply:
I referred to the Google Java Style Guide and did not see this style breaking rules. Although it is not very common but I think I will take it.

Let's all use the Google Java Formatter https://github.com/google/google-java-format#using-the-formatter to keep our format consistent.

Reply:
Thanks for the link, I was going to propose using [EditorConfig](https://editorconfig.org/) but maybe these are complementary. I'll take a look at the plugin you suggested.